Javascript/jq-test.js

```diff
@@ -1,69 +1,17 @@
-$( document ).ready(function() {
-    console.log( "ready!" );
```

Comment:
`$( document ).ready(function() {` is the same as `$(function() {`, so nesting it does nothing, I've just added one top level `$(function(){ ... }` which does the same thing